site/pages/about-us/q-and-a-slice/category/style.css

```diff
@@ -22,7 +22,8 @@
   width: 100%;
 }
 
-.category__questionList li:last-of-type h4 {
+.category__questionList li:last-of-type h4,
+.category__questionList li:last-of-type p {
   padding-bottom: 0;
```

Comment:
Hey @mveritym , the other h4 elements in the question list have a padding-bottom value of 20px. I think that also needs to be applied to the last question, as it's still aligned differently